PyU4V/tests/ci_tests/test_pyu4v_ci_replication.py

```diff
@@ -482,3 +483,111 @@ def test_failback_storage_group_srdf(self):
         status = self.replication.get_storage_group_srdf_details(
             storage_group_id=sg_name, rdfg_num=srdf_group_number).get('states')
         self.assertIn('Synchronized', status)
+
+    def test_get_rdf_director_list(self):
```

Comment:
nit, add a description.  It doesn't look like the version of our pep8 complains anymore but just to be consistent with the rest of the test file